README.md

```diff
@@ -65,12 +70,7 @@ Feel free to check the [issues page](../../issues/).
 
 Give a ⭐️ if you like this project!
 
-## Acknowledgments
-
-- Hat tip to anyone whose code was used
-- Inspiration
-- etc
 
 ## 📝 License
 
-This project is [MIT](./MIT.md) licensed.
+This project is [MIT](./MIT.md) licensed  and special credit to Cindy Shin (https://www.behance.net/adagio07) for this template under the Creative Commons license (https://creativecommons.org/licenses/by-nc/4.0/) .
```

Comment:
- [ ] You can do something like special credit to [Cindy Shin](https://www.behance.net/adagio07) for this template under the [Creative Commons license](https://creativecommons.org/licenses/by-nc/4.0/) to look more cleaner.